recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: holoviews
+  version: {{ version }}
+
+source:
+  fn: holoviews-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/h/holoviews/holoviews-{{version}}.tar.gz
+  md5: 195811299fcf1c4bff2cd413fb44970b
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - param
+    - setuptools
+  run:
+    - python
+    - param
+    - numpy
+    - matplotlib
+    - bokeh
+    - jupyter
+    - notebook
+    - ipython
+    - lancet
+
+test:
+  imports:
+    - holoviews
+
+about:
+  home: www.holoviews.org
+  summary: Stop plotting your data - annotate your data and let it visualize itself.
+  license: BSD 3-Clause
+
+
+extra:
+  recipe-maintainers:
+    - jlstevens
+    - philippjfr
+    - marqh
```

Comment:
@jlstevens just check with @philippjfr and @philippjfr if they want to be maintainers here. Everybody listed here will be automatically added to the `holoviews` recipe repository (feedstock) and will have commit rights there.